### DISCUSSION TITLE
Can't connect AWS RDS Aurora MySQL

### DISCUSSION THREAD

**Discussion by gardito-git** [gravitational/teleport#45406] [Q&A]
![image](https://github.com/user-attachments/assets/41fa60b5-ede1-41fb-a8ee-0e0daef4382b)

Hello everyone, I'm currently new in GoTeleport, I'm already success install GoTeleport, add Resources via Debian 8+ in multiple AWS Account and add Resources FrontEnd Containerized App by install GoTeleport and add custom teleport.yaml file inside Dockerfile. I'm trying to connect our existing databases in multiple account, this was my first but I can't connect/continues from GUI, can someone guide me ?

This is my goteleport IAM on 1 of our multiple account
![image](https://github.com/user-attachments/assets/04b6e8d0-38bb-4c57-9f92-a9e03021fdbd)
![image](https://github.com/user-attachments/assets/3bc212e9-b34a-49af-a33e-f4bcfcee8231)

**Comment by webvictim** [comment]
Hi - what version of Teleport are you running here? You can go to the top right, drop down the menu and click "Help and Support" to see - it'll be listed at the bottom of the page.

**Reply by gardito-git** [reply]
i'm running 16.1.0, Self-Hosted on Ubuntu EC2.
i'm sorry for late reply because I live on the east timezone, looks like i'm doing a fatal mistake, 
did I need another EC2 act as Proxy on each my multiple AWS Account, like bastion host ??

**Reply by Alex-Giaquinto** [reply]
I am currently facing the exact same issue when trying to enroll my EKS clusters. Do you have your teleport instance behind a VPN?

**Reply by marcoandredinis** [reply]
A single Proxy can handle multiple AWS Accounts 👍 
We usually recommend an integration per AWS Account, cross account access can work but is not well-tested.
Looking at the error above, it seems to fail to access the public keys. The public keys are stored in your S3 bucket. Can you please see if you can publicly (and without any authentication) those keys?

Alternatively, and possibly a better approach, would be to create a new Integration and not set any S3 fields.
Your current version (v16.1.0) has them as optional, but later versions do not even allow those fields.
